CMakeLists.txt

```diff
@@ -421,6 +421,7 @@ if(ICB)
   add_test(iso_c_binding_c Tests/iso_c_binding_c)
   set(CHECK_EXE "${CHECK_EXE} iso_c_binding_c")
 
+  enable_language(CXX)
```

Comment:
I don't think we want that to be unconditionally set.
I guess most of the users of arpack won't enable this option, we should not force them to need a C++ compiler then.

Reply:
By the way, this is not unconditionally set: this is conditioned by a "if(ICB)". ICB must be turned on at cmake time. if on, one need a c++ compiler for testing iso_c_binding for c++ (I added 2 tests: one for fortran/c binding, and, another one for fortran/c++ binding)